pkg/forwarder/sync_forwarder.go

```diff
@@ -41,7 +41,12 @@ func (f *SyncForwarder) Stop() {
 func (f *SyncForwarder) sendHTTPTransactions(transactions []*transaction.HTTPTransaction) error {
 	for _, t := range transactions {
 		if err := t.Process(context.Background(), f.client); err != nil {
-			log.Errorf("SyncForwarder.sendHTTPTransactions: %s", err)
+			log.Debugf("SyncForwarder.sendHTTPTransactions first attempt: %s", err)
```

Comment:
I changed this to a DEBUG log to reduce noise in the logs. By default, we will only log an error if the second/final attempt fails.